generator/src/generator.rs

```diff
@@ -679,7 +676,8 @@ mod tests {
                         state.match_string("d")
                     })
                 })
-            }.to_string()
+            }
```

Comment:
Is there a way to refactor this to something more manageable?

Reply:
For the contents of `quote!`? Not really, not while maintaining the "obvious correctness" of what's being tested against. For the fact `to_string` is on its own line? Unless we write a routine to compare `TokenStream` directly, this is the best we'll get. Of course, the great thing about rustfmt is not having to worry about minor formatting things and just asking rustfmt to do it for you.

(It formats within `mac!(..)` and treats it like a normal function call, but within `mac! { .. }` you're on your own for formatting.)